contracts/governance/Staking/Staking.sol

```diff
@@ -298,21 +298,120 @@ contract Staking is
     }
 
     /**
-     * @notice Withdraw tokens for vesting contract.
-     * @param vesting The address of Vesting contract.
+     * @notice Withdraw the given amount of tokens.
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
      * @param receiver The receiver of the tokens. If not specified, send to the msg.sender
-     * @dev Can be invoked only by whitelisted contract passed to governanceWithdrawVesting.
+     * @param vestingConfig The vesting config.
+     * @dev VestingConfig struct intended to avoid stack too deep issue, and it contains this properties:
+        address vestingAddress; // vesting contract address
+        uint256 startDate; //start date of vesting
+        uint256 endDate; // end date of vesting
+        uint256 cliff; // cliff. After this time period the tokens begin to unlock.
+        uint256 duration; // The duration. After this period all tokens will have been unlocked.
+        address tokenOwner; // The owner of the vested tokens.
+     * @dev Only used to governance direct withdrawal.
```

Comment:
```suggestion
        address vestingAddress; // vesting contract address
        uint256 startDate; //start date of vesting
        uint256 endDate; // end date of vesting
        uint256 cliff; // after this time period the tokens begin to unlock
        uint256 duration; // after this period all the tokens will be unlocked
        address tokenOwner; // owner of the vested tokens
    * @dev used exclusively for governance direct withdrawal
```

Reply:
fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/c4c43c22e2dc30a936960e01e9a8acd6e2053616)